ext/drpttracer/drpttracer.dox

```diff
@@ -88,37 +85,48 @@ the buffer size is 2^pt_size_shift * PAGE_SIZE.
  - sideband_size_shift: The size shift of PT sideband data's ring buffer. It must be
 greater than 0, and the buffer size is 2^sideband_size_shift * PAGE_SIZE.
 
-\note Linux perf sets the buffer size to 4MiB by default. Therefore, it is best for clients
-to set trace and sideband buffers larger than 4Mib.
+\note Linux perf sets the buffer size to 4MiB by default. Therefore, it is best for
+clients to set trace and sideband buffers larger than 4Mib.
+
+\note For one thread, only one tracing can execute at the same time. So the client needs
+to ensure one thread only owns one pttracer handle.
 
 The client must ensure that the PT trace's ring buffer and the sideband data's ring buffer
 is large enough to hold the PT data. Insufficient buffer size will lead to lost data,
-which may cause issues in pt2ir_t decoding. After the tracing is started, the
-client can get  a handle for the PT tracing. The handle is a pointer to \p drpttracer's
-internal data structure which contains the tracing's PT data buffer, sideband data buffer,
-and metadata.
-
-To stop collecting the trace, the client needs to pass the trace handle to
-drpttracer_end_tracing(). When the tracing is stopped, the stop function will allocate an
-instance of #drpttracer_output_t, and copy the PT data, sideband data, and metadata to a
-#drpttracer_output_t instance which is returned to the instance to the client. The client
-can dump the data in the output to different files for offline post-processing. The user
-can use the library pt2ir_t in drcachesim to convert the PT trace to DynamoRIO's
-IR.
-
-Additionally, if drpttracer_end_tracing() detect an overflow, it will return an error
+which may cause issues in pt2ir_t decoding.
+
+After creating a pttracer handle successfully, the client can start or stop a tracing
+session by calling drpttracer_start_tracing() and drpttracer_stop_tracing().
+
+ - drpttracer_start_tracing()
+ - drpttracer_stop_tracing()
+
+The client must pass an invaild trace handle to the start and stop function. And the
```

Comment:
"invalid" is spelled wrong: but I don't think that's the word you mean -- noone should pass an invalid anything.  Do you mean "valid"?

Reply:
Done.